pybamm/input/parameters/lithium_ion/separators/separator_Xu2019/README.md

```diff
@@ -0,0 +1,7 @@
+# Separator parameters
+
+Parameters for a ??? separator, from the paper
```

Comment:
Do you want to leave the ??? or was that a placeholder?

Reply:
placeholder, good spot